app/assets/stylesheets/hyrax/_work-show.scss

```diff
@@ -56,3 +56,28 @@ ul.tabular {
   margin-bottom: 18px;
   margin-top: -8px;
 }
+
+.panel-workflow {
+  background: #fff;
+  border-color: #e0e0e0;
+  left: 0;
```

Comment:
Why is this needed?  Is there some reason we aren't using or inheriting from the bootstrap `panel-default` class?

Reply:
`panel-default` is used by the main panel, so I think overriding it would cause a problem. I think overriding any of the bootstrap panels provided is likely to cause confusion down the line if they are used on that page (or the style is picked up accidentally) and this makes the panel styling explicit for our use case. I'm open to a suggestion on a best practice that shows this approach is problematic though :-)

Reply:
Okay, I'm fine with the class name. I didn't notice that this is still a `.panel`. Is the `left: 0`, `margin: 0 auto`, `max-width: 1140px` still required? Those seem like properties you'd set on fixed position elements, but not on inline elements.

 How about the background-color? Aren't the panel-heading and panel-body setting a background color that will override this one?